pyvista/plotting/render_passes.py

```diff
@@ -0,0 +1,255 @@
+"""Render classes module for PyVista."""
+import weakref
+
+from pyvista import _vtk
+
+# ordered
+PRE_PASS = [
+    'vtkEDLShading',
+]
+
+# ordered
+POST_PASS = [
+    'vtkDepthOfFieldPass',
+    'vtkGaussianBlurPass',
+    'vtkOpenGLFXAAPass',
+    'vtkSSAAPass',  # should really be last
+]
+
+
+class RenderPasses:
+    """Class to support multiple render passes for a renderer.
+
+    Notes
+    -----
+    Passes are organized here as "Primary" (vtkOpenGLRenderPass) that act
+    within the renderer and "post-processing" (vtkImageProcessingPass) passes,
+    which act on the image generated from the renderer.
+
+    The primary passes are added as part of a vtk.vtkRenderPassCollection or
+    are "stacked", while the post-processing passes are added as a final pass
+    to the rendered image.
+
+    """
+
+    def __init__(self, renderer):
+        """Initialize render passes."""
+        self._renderer_ref = weakref.ref(renderer)
+
+        self._passes = {}
+        self._fxaa_pass = None
+        self._shadow_map_pass = None
+        self._edl_pass = None
+        self._dof_pass = None
+        self._ssaa_pass = None
+        self._blur_passes = []
+        self.__pass_collection = None
+        self.__seq_pass = None
+        self.__camera_pass = None
+
+    @property
+    def _pass_collection(self):
+        """Initialize (when necessary the pass collection and return it.
+
+        This lets us lazily generate the pass collection only when we need it
+        rather than at initialization of the class.
+
+        """
+        if self.__pass_collection is None:
+            self._init_passes()
+        return self.__pass_collection
+
+    @property
+    def _seq_pass(self):
+        """Initialize (when necessary the pass collection and return it.
+
+        This lets us lazily generate the pass collection only when we need it
```

Comment:
"pass collection" was copied over here. I don't know what "seq_pass" exactly means, so I won't try to fix this.

Reply:
Fixed in 69243a9ff453f7d292ce9329026b0ef8c43fa355.